tools/bin/ci_check_dependency.py

```diff
@@ -17,9 +17,20 @@
     "/integration_tests/", "/unit_tests/",
     # Common
     "acceptance-test-config.yml", "acceptance-test-docker.sh", ".md", ".dockerignore", ".gitignore", "requirements.txt"]
+IGNORED_SOURCES = [
+    re.compile("^source-e2e-test-cloud$"),
+    re.compile("^source-mongodb$"),
```

Comment:
FYI `source-mongodb-v2` is the maintained MongoDb source. The other one is legacy.